src/View/Definition.elm

```diff
@@ -52,6 +50,7 @@ view model =
                     ]
                 ]
             ]
+        , verticalSpacing 2
```

Comment:
Did we want this in the page tpl in Main?

Reply:
... that makes sense